src/api/spec/controllers/webui/configuration_controller_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe Webui::ConfigurationController do
+  let(:confirmed_user) { create(:confirmed_user) }
+  let(:admin_user) { create(:admin_user) }
+
+  describe 'GET #index' do
+    it { should use_before_action(:require_admin) }
+
+    it 'is allowed as admin User' do
```

Comment:
same here, why do you test this? Don't you trust your own `:require_admin` test? ;)